src/lib/skilling/skills/construction/cuttables.ts

```diff
@@ -0,0 +1,36 @@
+import itemID from '../../../util/itemID';
+import { Cuttable } from '../../types';
+
+const Plankables: Cuttable[] = [
+	{
+		name: 'Logs',
+		inputItem: itemID('Logs'),
+		outputItem: itemID('Plank'),
+		gpCost: 100
+	},
+	{
+		name: 'Oak',
+		inputItem: itemID('Oak logs'),
+		outputItem: itemID('Oak plank'),
+		gpCost: 250
+	},
+	{
+		name: 'Teak',
+		inputItem: itemID('Teak logs'),
+		outputItem: itemID('Teak plank'),
+		gpCost: 500
+	},
+	{
+		name: 'Mahogany',
+		inputItem: itemID('Mahogany logs'),
+		outputItem: itemID('Mahogany plank'),
+		gpCost: 1500
+	}
+];
+
+const Cuttables = {
+	aliases: ['saw'],
+	Plankables
+};
+
+export default Cuttables;
```

Comment:
Why are Plankables a property of Cuttables? this makes no sense.  You should structure it like other skills, however its probably a better idea to just have these in `lib/minions/data`, since we dont have construction yet, so having an incomplete skill here would be just confusing/inconsistent.